src/detect-lua.h

```diff
@@ -56,6 +56,7 @@ typedef struct DetectLuaData {
     uint64_t alloc_limit;
     uint64_t instruction_limit;
     int allow_restricted_functions;
+    lua_State *luastate;
```

Comment:
Should this be here? For Lua detect script the state is in `DetectLuaThreadData`.

Reply:
I'm not sure yet how to get access to the det_context from a transform so I temporarily did this.. Thoughts?

Reply:
Think the transform API just needs to be updated to pass `det_ctx` around.